app/models/post.rb

```diff
@@ -2,18 +2,29 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
+
+  before_validation :set_comments_counter, on: :create
+  before_validation :set_likes_counter, on: :creat
```

Comment:
![image](https://github.com/fithamlak/blog-app/assets/93039359/74ac4709-a6b2-40b6-ad2d-8b7bc41bbf21)

**[OPTIONAL]**

- [ ] As you can see in the image above that I am not able to create a post on your app because you have a typo in the post.rb model line 7 😨 You have written `creat` instead of `create` kindly fix this typo to make your form work as expected 👍